src/main/java/core/basesyntax/StorageImpl.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+public class StorageImpl<K, V> implements Storage<K, V> {
+    static final int MAX_NUMBER = 10;
+    private final Object[] keyArray = new Object[MAX_NUMBER];
+    private final Object[] valueArray = new Object[MAX_NUMBER];
```

Comment:
you should initialize these fields in constructor, like this: `keyArray = (K[]) new Object[MAX_NUMBER];`
```suggestion
    private final K[] keyArray;
    private final V[] valueArray;
```